src/main/java/minijava/ir/optimize/AliasAnalyzer.java

```diff
@@ -824,6 +837,7 @@ public boolean transform() {
       Node ptr = sideEffect.getPred(1);
       Set<IndirectAccess> aliasClass = getPointsTo(ptr);
       boolean isLoad = sideEffect.getOpCode() == iro_Load;
+      assert !aliasClass.isEmpty() : "Empty alias class at " + sideEffect;
```

Comment:
Right, @parttimenerd enabled this here: https://github.com/mj3-16/minijavac/blame/ffe577850cc820dc72c32f140124f352938e1967/build.gradle#L46
Was probably in one of the PRs I didn't really review ;)
Enabling it always by default might not be the best idea, though. I'm not sure how big the runtime cost is, but later in the competition this might make the difference ;)